File: aspnet-core/src/doan.ProjectManagement.Application.Contracts/TeacherInformationGroups/Dto/TeacherInformationGroupDto.cs
using System;
using Volo.Abp.Application.Dtos;

namespace doan.ProjectManagement.TeacherInformationGroups.Dto
{
    public class TeacherInformationGroupDto : EntityDto<Guid>
    {
        public Guid? TeacherId { get; set; }
        public string TeacherName { get; set; }
        public string Roles { get; set; }
        public string Description { get; set; }
        public Guid? ProjectInformationId { get; set; }
    }
}

File: applications/admin-web/src/app/shared/services/teacher/teacher-information-group/models/teacher-information-group-dto.ts
import { EntityDto } from '@abp/ng.core';

export class TeacherInformationGroupDto extends EntityDto<string>{
  teacherId: string;
  teacherName: string;
  roles: string;
  description: string;
  projectInformationId:string;

  constructor(initial:Partial<TeacherInformationGroupDto> = {}){
    super(initial)
  }
}
